amplify-tools-gradle-plugin/src/main/resources/META-INF/gradle-plugins/com.amplifyframework.amplifytools.properties

```diff
@@ -0,0 +1 @@
+implementation-class=AmplifyTools
```

Comment:
This file name, `com.amplifyframework.amplifytools.properties `-- the import path needs to be consistent with the rest of the PR (for example, what you use in the manifest.)